collate/collate.py

```diff
@@ -7,6 +7,23 @@
 from .sql import make_sql_clause, to_sql_name, CreateTableAs, InsertFromSelect
 
 
+def execute_insert(get_engine, insert):
```

Comment:
What's the destiny for `ka/parallel`? We need to refactor things if we want to merge it into master eventually anyway.